libs/tktokenswap/test/TestUtils/DecodedProblemData.cpp

```diff
@@ -0,0 +1,179 @@
+// Copyright 2019-2022 Cambridge Quantum Computing
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "DecodedProblemData.hpp"
+
+#include <catch2/catch_test_macros.hpp>
+#include <tktokenswap/GeneralFunctions.hpp>
+#include <tktokenswap/VertexSwapResult.hpp>
+
+using std::vector;
+
+namespace tket {
+namespace tsa_internal {
+namespace tests {
+
+/// TODO: move somewhere more appropriate.
```

Comment:
Is this todo documented somewhere?

Reply:
No, but see above (this comment existed already). Probably this could do with some refactoring.